tests/UDQ_ACTIONX_TEST1.DATA

```diff
@@ -459,6 +466,190 @@ WELOPEN
 /
 ENDACTIO
 
+-- ACTIONX - test data
```

Comment:
I guess commented out test data can just be removed?

Reply:
The commented part of test data set is deleted.